packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/OverviewTabPanel.tsx

```diff
@@ -249,3 +299,105 @@ const RemindersCard = ({
+}: {
+  creditNotes: components['schemas']['ReceivableResponse'][];
+  internalNavigation?: boolean;
+}) => {
+  const { i18n } = useLingui();
+
+  const transformCreditNotes = (
+    creditNotes: components['schemas']['ReceivableResponse'][]
+  ): TransformCreditNotes[] => {
+    if (!creditNotes) return [];
+
+    return creditNotes.map((creditNote) => {
+      const issueDate = creditNote.issue_date
+        ? new Date(creditNote.issue_date)
+        : null;
+
+      const formattedDate = issueDate
+        ? i18n.date(issueDate, DateTimeFormatOptions.EightDigitDate)
+        : t(i18n)`Unknown date`;
+
+      const authorName =
+        creditNote.entity.type !== 'individual' && creditNote.entity.name
+          ? creditNote.entity.name
+          : creditNote.entity.type !== 'organization'
+          ? `${creditNote.entity.first_name} ${creditNote.entity.last_name}`
+          : null;
+
+      return {
+        title: creditNote.document_id,
+        description: `${t(i18n)`Issued on`} ${formattedDate} ${t(i18n)`by`}`,
+        authorTitle: authorName || '',
+      };
+    });
+  };
+
+  const data = transformCreditNotes(creditNotes);
+
+  if (!data || data.length === 0) return null;
+
+  return (
+    <Card
+      sx={{ borderRadius: 3, bgcolor: 'background.paper', px: 2 }}
+      variant="outlined"
+    >
+      <Grid container direction="column">
+        {data?.map((item, index) => (
+          <Grid
+            key={item.title}
+            container
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+            sx={{
+              py: 1.5,
+              ...(index
+                ? { borderTop: '1px solid', borderTopColor: 'divider' }
+                : {}),
+              cursor: 'pointer',
+            }}
+          >
+            <Grid item container direction="column" xs>
+              <Typography
+                variant="body1"
+                fontWeight="bold"
+                sx={{ textTransform: 'capitalize' }}
+              >
+                {item.title}
+              </Typography>
+              <Typography variant="body2">
+                {item.description}{' '}
+                {item.authorTitle ? (
+                  // Only render as a link if internal navigation is available
+                  internalNavigation ? (
+                    <Link
+                      href="#" // Placeholder link, should be replaced when interlinking is available
+                      underline="hover"
+                      color="primary"
+                      variant="body2"
+                    >
+                      {item.authorTitle}
+                    </Link>
+                  ) : (
+                    item.authorTitle
+                  )
+                ) : null}
+              </Typography>
+            </Grid>
+            <Grid item>
+              <IconButton edge="end" size="small">
+                <ArrowForwardIcon fontSize="small" />
```

Comment:
As long as we don't have internal navigation, we should neither show arrows nor add `cursor: 'pointer'`.
Release from `dev` to prod should be possible at any time.
There's no need to mislead users.